### ISSUE TITLE
Windows image loading code

### ISSUE THREAD

**Issue by Hexadecimaaal** [ice1000/jimgui#6]
```cpp
#include <d3dx9tex.h>

auto CreateTextureFromFile(const char *fileName, int &width, int &height) -> LPDIRECT3DTEXTURE9
{
    LPDIRECT3DTEXTURE9 t;
    size_t size = strlen(fileName) + 1;  
    wchar_t* wFileName = new wchar_t[size]; 

    size_t outSize;
    mbstowcs_s(&outSize, wFileName, size, fileName, size-1);
    auto hr = D3DXCreateTextureFromFile(g_pd3dDevice, wFileName, &t);
    D3DSURFACE_DESC desc;
    t->GetLevelDesc(0, &desc);
    width = static_cast<int>(desc.Width);
    height = static_cast<int>(desc.Height);
    if (SUCCEEDED(hr)) return t;
    else return nullptr;
}
```

**Comment by ice1000** [comment]
Oh no

**Comment by ice1000** [comment]
Closed by #7
